### PR TITLE
Add std feature

### PR DESCRIPTION
**Pull request by leifhelm**
It seems to me that a few thing were missing to have some convenience features with the standard library

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #![no_std]
 #![deny(missing_debug_implementations)]
 
+#[cfg(feature = "std")]
```
Comment (reitermarkus) [inline src/lib.rs:4]: This should also disable the `no_std` attribute.

File: Cargo.toml [hunk 2]
```diff
@@ -18,3 +18,7 @@ derive-try-from-primitive = "1.0"
 mbusparse = { version = "0.1" }
 nom = "7.1"
 serde = { version = "1.0", features = ["derive"], optional = true }
```
Comment (reitermarkus) [inline Cargo.toml:20]: Add `default-features = false` here.

File: Cargo.toml [hunk 3]
```diff
@@ -18,3 +18,7 @@ derive-try-from-primitive = "1.0"
...
+
+[features]
+default = []
+std = []
```
Comment (reitermarkus) [inline Cargo.toml:24]:
```suggestion
std = ["serde/std"]
```

File: Cargo.toml (hunk 2 above)
Comment (leifhelm) [reply Cargo.toml]: I added `default-features = false` to all crates needing it

**Comment by reitermarkus** [comment]
Thanks, @leifhelm!

**Pull request merged**